testing/armagetronad/APKBUILD

```diff
@@ -0,0 +1,45 @@
+# Maintainer: Díaz Urbaneja Diego <sodomon2@gmail.com>
+pkgname=armagetronad
+pkgver=0.2.8.3.4
+pkgrel=0
+pkgdesc="A Tron Clone in 3D."
+url="http://armagetronad.net/"
+arch="x86 x86_64"
+license="GPL"
```

Comment:
This isn't a SPDX compliant license. See https://spdx.org/licenses/ for a list of valid license expressions.